packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -643,6 +646,77 @@ export class AccountHandler {
     }
   }
 
+  async setPassword(request: AuthRequest) {
+    this.log.begin('Account.set_password', request);
```

Comment:
nit: I think this should be 'Account.setPassword' to be consistent the other log.begin calls.

Reply:
In the off chance this can make it into the sprint, I'll fix this in my next PR if that's okay!